### PR TITLE
[DOCS] checkpoint cron howto

### PR DESCRIPTION
**Pull request by Aylr**
Changes proposed in this pull request:
- checkpoint cron howto
-
-

Previous Design Review notes:
- 
-
-

Thank you for submitting!

### CODE REVIEW

File: docs/how_to_guides/validation/how_to_deploy_a_scheduled_checkpoint_with_cron.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
+###################################################
+How to deploy a scheduled Checkpoint with cron
+####################################################
+
+In this guide you'll learn how to deploy a `Checkpoint <link here>_` on a schedule using cron.
+
+.. admonition:: Prerequisites
+
+    This article assumes that in your deployment environment you have access to:
```
Comment (abegong) [inline docs/how_to_guides/validation/how_to_deploy_a_scheduled_checkpoint_with_cron.rst]: Please add the standard blurb about having already configured a working deployment. I want to habituate users to understand that that's the royal road to getting real stuff done in GE.

File: docs/how_to_guides/validation/how_to_deploy_a_scheduled_checkpoint_with_cron.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
+###################################################
+How to deploy a scheduled Checkpoint with cron
+####################################################
+
+In this guide you'll learn how to deploy a `Checkpoint <link here>_` on a schedule using cron.
+
+.. admonition:: Prerequisites
```
Comment (abegong) [inline docs/how_to_guides/validation/how_to_deploy_a_scheduled_checkpoint_with_cron.rst]: Please use the standard header here

File: docs/how_to_guides/validation/how_to_deploy_a_scheduled_checkpoint_with_cron.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
+###################################################
+How to deploy a scheduled Checkpoint with cron
+####################################################
+
+In this guide you'll learn how to deploy a `Checkpoint <link here>_` on a schedule using cron.
```
Comment (abegong) [inline docs/how_to_guides/validation/how_to_deploy_a_scheduled_checkpoint_with_cron.rst]: Minor nit, how-tos aren't about teaching or learning. "This guide will show you how to..." "This guide will walk you through the steps..." "To {do X}, follow the steps in this guide..."

**Review by abegong — CHANGES_REQUESTED** [review]
The content is great, but it doesn't follow the new structure and template for how-to guides:
* https://docs.greatexpectations.io/en/latest/how_to_guides/miscellaneous/how_to_write_a_how_to_guide.html
* https://docs.greatexpectations.io/en/latest/how_to_guides/miscellaneous/how_to_template.html

Not including numbered steps is an especially big departure.

I want to be rigorous about following a prescribed format, since (1) it'll keep us on the straight and narrow for writing "how-to" guides, and not wandering into reference material, (2) once we hit a rhythm, it'll help us knock out guides faster, and (3) when contributors start writing guides, it'll help us enforce greater consistency.

File: docs/how_to_guides/validation/how_to_deploy_a_scheduled_checkpoint_with_cron.rst (hunk 2 above)
Comment (Aylr) [reply docs/how_to_guides/validation/how_to_deploy_a_scheduled_checkpoint_with_cron.rst]: Ah yep - this PR predates my knowledge of the template. Fixing

**Comment by abegong** [comment]
Minor changes:
* Capitalized "Checkpoint", since it's a core concept. Ideally, we'd also link on first reference, but that reference article hasn't been written yet.
* Fixed an ungrammatical sentence.

**Pull request merged**